BP/common/religion/holy_sites/BP_holy_sites.txt

```diff
@@ -1,76 +1,354 @@
+}
+sinope = {
+	county = c_sinope
+	character_modifier = {}
+}
+trebizond = {
+	county = c_trebizond
+	character_modifier = {}
+}
+#############
+# Georgian Pagan
+#############
+abkhazia = {
+	county = c_abkhazia
+	character_modifier = {}
+}
+ani = {
+	county = c_hayk
+	barony = b_ani
+	character_modifier = {}
+}
+shirvan = {
+	county = c_shirvan
+	character_modifier = {}
+}
+#############
+# Dacian Pagan
+#############
+mesembria = {
+	county = c_mesembria
+	character_modifier = {}
+}
+giurgiu = {
+	county = c_giurgiu
+	character_modifier = {}
+}
+pest = {
+	county = c_visegrad
+	barony = b_pest
+	character_modifier = {}
+}
+bereg = {
+	county = c_bereg
+	character_modifier = {}
+}
+chilia = {
+	county = c_chilia
+	character_modifier = {}
+}
+#############
+# Scythian Pagan
+#############
+itil = {
+	county = c_itil
+	character_modifier = {}
+}
+dihistan = {
+	county = c_dihistan
+	character_modifier = {}
+}
+#############
+# Semitic Pagan
+#############
+muscat = {
+	county = c_muscat
+	character_modifier = {}
+}
+#############
+# Gurzil Pagan
+#############
+garama = {
+	county = c_germa
+	character_modifier = {}
+}
+trapani = {
+	county = c_palermo
+	barony = b_trapani
+	character_modifier = {}
+}
+leptis_magna = {
+	county = c_labda
+	character_modifier = {}
+}
+#############
+# Celtic Pagan
+#############
+vannes = {
+	county = c_vannes
+	character_modifier = {}
+}
+isle_of_man = {
+	county = c_isle_of_man
+	character_modifier = {}
+}
+gowrie = {
+	county = c_gowrie
+	character_modifier = {}
+}
+#############
+# Germanic Pagan
+#############
+magdeburg = {
+	county = c_magdeburg
+	character_modifier = {}
+}
+york = {
+	county = c_north_riding
+	character_modifier = {}
+}
+bregenz = {
+	county = c_ravensburg
+	barony = b_bregenz
+	character_modifier = {}
+}
+#############
+# Egyptian Pagan
+#############
+rosetta = {
+	county = c_al-buhaira
+	barony = b_rashid
+	character_modifier = {}
+}
+gizeh = {
+	county = c_giza
+	barony = b_gizeh
+	character_modifier = {}
+}
+asyut = {
+	county = c_asyut
+	character_modifier = {}
+}
+#############
+# Basque Pagan
 #############
 toulouse = {
 	county = c_toulouse
-
-	character_modifier = {
-		name = holy_site_mecca_effect_name
-		monthly_piety_gain_per_knight_mult = 0.02
-		knight_effectiveness_mult = 0.1
-	}
-}
\ No newline at end of file
+	character_modifier = {}
+}
+cadiz = {
+	county = c_cadiz
+	character_modifier = {}
+}
+asturias_de_santillana = {
+	county = c_asturias_de_santillana
+	character_modifier = {}
+}
+navarra = {
+	county = c_navarra
+	character_modifier = {}
+}
+#############
+# Greco-Roman Pagan
+#############
+mt_olympus = {
+	county = c_veria
+	barony = b_kalyvia
+	character_modifier = {}
+}
+olympia = {
+	county = c_messenia
+	barony = b_arcadia
+	character_modifier = {}
+}
+pangaion_hills = {
+	county = c_serres
+	barony = b_chrysopolis
+	character_modifier = {}
+}
+pimpleia = {
+	county = c_veria
+	barony = b_veria
+	character_modifier = {}
+}
+cronton = {
```

Comment:
It should be Kroton or Crotona, I think you made a typo here

Reply:
Fixed.

Reply:
There is still an issue, there is an "N" too much, you wrote "Crontona" as opposed to "Crotona"

Reply:
Fixed again. Apparently I am incapable of spelling.